flexget/plugins/clients/qbittorrent.py

```diff
@@ -261,6 +268,16 @@ def add_entries(self, task, config):
             if maxdownspeed:
                 form_data['dlLimit'] = maxdownspeed * 1024
 
+            ratio_limit = entry.get('ratio_limit', config.get('ratio_limit'))
+            if ratio_limit:
+                form_data['ratioLimit'] = ratio_limit
+
+            seeding_time_limit = entry.get('seeding_time_limit', config.get('seeding_time_limit'))
+            if seeding_time_limit:
+                form_data['seedingTimeLimit'] = int(
+                    parse_timedelta(seeding_time_limit).total_seconds() / 60
```

Comment:
Hmm, when this ends up as the default of -1, is this going to fail? Maybe take away the default, and check if the option `is None` here, and in that case set it to -1 instead of trying to parse.

Reply:
That's a good point actually... both options are optional.